fission-server/src/models/account.rs

```diff
@@ -195,3 +200,64 @@ impl From<Account> for AccountRequest {
         }
     }
 }
+
+/// Account with Root Authority (UCAN)
+#[derive(Clone, Debug)]
+pub struct RootAccount {
+    /// The Associated Account
+    pub account: Account,
+    /// A UCAN with Root Authority
+    pub ucan: ucan::Ucan,
```

Comment:
Rather than manually implementing `Serialize`, you might find it easier to use `serialize_with` to override the serialization function for this field, and then derive `Serialize` for the `RootAccount` struct.. You'll still need to convert the errors, but won't need to implement and keep everything else in sync.

https://serde.rs/field-attrs.html#serialize_with